go/chat/push.go

```diff
@@ -287,10 +290,12 @@ func (g *PushHandler) Activity(ctx context.Context, m gregor.OutOfBandMessage, b
 			badger.PushChatUpdate(*nm.UnreadUpdate, nm.InboxVers)
 		}
 	default:
-		return fmt.Errorf("unhandled chat.activity action %q", action)
+		err = fmt.Errorf("unhandled chat.activity action %q", action)
+		return err
```

Comment:
pretty sure this doesn't have to be two lines and that `return fmt.Errorf(...` will still set `err`